test/components/views/messages/TextualBody-test.js

```diff
@@ -41,6 +42,7 @@ describe("<TextualBody />", () => {
             isGuest: () => false,
             mxcUrlToHttp: (s) => s,
         };
+        DMRoomMap.makeShared();
```

Comment:
Can you move this into a `beforeEach(() =>...);` above the afterEach instead?

Reply:
Unfortunately no, because `DMRoomMap.makeShared` depends on `MatrixClient`, and this test nulls out then sets up `MatrixClient` manually on just about every test case.

Reply:
Ah fair enough